contracts/farm/StakingPool.sol

```diff
@@ -502,6 +522,34 @@ contract StakingPools is ReentrancyGuard {
     return myreferees[_poolId][referral];
   }
 
+  /// @dev To get withdrawable amount that has passed lockup period of a pool
+  function getWithdrawAbleAmount(uint256 _poolId, address _account) public view returns (uint256) {
```

Comment:
It seems a typo error - "getWithdrawAbleAmount" should be "getWithdrawableAmount"